tests/rest/RstcheckBearTest.py

```diff
@@ -0,0 +1,18 @@
+from bears.rest.RstcheckBear import RstcheckBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file1 = "test\n====\n"
+bad_file1 = "test\n==\n"
+
+good_file2 = "=====\nTest0\n=====\n.. code-block:: python\n\n    print()"
+bad_file2 = "=====\nTest0\n=====\n.. code-block:: python\n\n    print(\n"
+
+good_file3 = "====\ntest\n====\n=====\ntest2\n=====\n"
+bad_file3 = "====\ntest\n====\n====\ntest\n====\n"
```

Comment:
can you give the tests meaningful names inttead of numbering them? You have to cope with people like me who can't keep track of those things ;) every test should have a purpose, if a test doesn't have a purpose, you can remove it. I actually think you might remove lines 6 to 11 but you'll know that better than me.